app/Http/Controllers/HomeController.php

```diff
@@ -120,11 +125,13 @@ public function getPhone($phone)
                 'phone' => $phone,
             ];
             $this->user->update($id, $data);
+            $this->usermeta->updateDisplayPhone($id, $radio);
 
             return response()->json([
                 'data' => '1',
             ]);
         } else {
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Squiz.WhiteSpace.ControlStructureSpacing.SpacingAfterOpen</br>**Severity**: error</br>**File**: app/Http/Controllers/HomeController.php</br>**Line**: 133</br>Blank line found at start of control structure